eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -82,40 +75,18 @@ parameters:
   - name: AdvancedBuild
     type: boolean
     default: false
+  - name: oneESTemplateTag
+    type: string
+    default: release
 
 extends:
-  ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-    template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
-  ${{ else }}:
-    template: v1/1ES.Unofficial.PipelineTemplate.yml@1ESPipelineTemplates
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
   parameters:
-    settings:
-      skipBuildTagsForGitHubPullRequests: true
-    sdl:
-      sourceAnalysisPool:
-        name: azsdk-pool-mms-win-2022-general
-        image: azsdk-pool-mms-win-2022-1espt
-        os: windows
-      eslint:
-        enabled: false
-        justificationForDisabling: "ESLint injected task has failures because it uses an old version of mkdirp. We should not fail for tools not controlled by the repo. See: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3556850"
-      codeql:
```

Comment:
Are these not needed? As I don't see it 1es-redirect?

Reply:
Oh might have been a mismatch/gap, thanks for catching.

Reply:
Unless @scbedd when you added the initial 1es-redirect you intended to remove these?

Reply:
ESLint and codeql are definitely necessary. 

Git history says I was the one to add it, but I definitely don't recall using 1es redirect for any of the python yml. No idea how that happened.